packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js

```diff
@@ -524,13 +527,36 @@ QUnit.module('decorators markup', {}, () => {
             selectAllText: 'Test'
         }));
 
-        const $multipleContainer = $list.find('.dx-list-select-all');
+        const $multipleContainer = $list.find(toSelector(SELECT_ALL_CLASS));
         assert.equal($multipleContainer.length, 1, 'container for SelectAll rendered');
         assert.equal($multipleContainer.text(), 'Test', 'select all rendered');
         const $checkbox = $multipleContainer.find('.dx-checkbox');
         assert.equal($checkbox.length, 1, 'checkbox rendered');
     });
 
+    QUnit.test('selectAll text and aria-label attribute should be equal to custom localized text (T1239880)', function(assert) {
+        localization.loadMessages({ 'en': { 'dxList-selectAll': 'custom-select-all' } });
+        localization.locale('en');
```

Comment:
`en` is a default locale, so no need to do it
```suggestion
```

Reply:
[b71911d](https://github.com/DevExpress/DevExtreme/pull/27664/commits/b71911da4022244261cdc0aa630106f28be9cee3)